Sources/Subs-Compat.php

```diff
@@ -24,28 +33,4918 @@
+	}
+	function ssi_recentPoll($topPollInstead = false, $output_method = 'echo'): ?array
+	{
+		return SMF\ServerSideIncludes::recentPoll($topPollInstead, $output_method);
+	}
+	function ssi_showPoll(?int $topic = null, string $output_method = 'echo'): ?array
+	{
+		return SMF\ServerSideIncludes::showPoll($topic, $output_method);
+	}
+	function ssi_pollVote()
+	{
+		return SMF\ServerSideIncludes::pollVote();
+	}
+	function ssi_quickSearch(string $output_method = 'echo'): ?string
+	{
+		return SMF\ServerSideIncludes::quickSearch($output_method);
+	}
+	function ssi_news(string $output_method = 'echo'): ?string
+	{
+		return SMF\ServerSideIncludes::news($output_method);
+	}
+	function ssi_todaysBirthdays(string $output_method = 'echo'): ?array
+	{
+		return SMF\ServerSideIncludes::todaysBirthdays($output_method);
+	}
+	function ssi_todaysHolidays(string $output_method = 'echo'): ?array
+	{
+		return SMF\ServerSideIncludes::todaysHolidays($output_method);
+	}
+	function ssi_todaysEvents(string $output_method = 'echo'): ?array
+	{
+		return SMF\ServerSideIncludes::todaysEvents($output_method);
+	}
+	function ssi_todaysCalendar(string $output_method = 'echo'): ?array
+	{
+		return SMF\ServerSideIncludes::todaysCalendar($output_method);
+	}
+	function ssi_boardNews(
+		?int $board = null,
+		?int $limit = null,
+		?int $start = null,
+		?int $length = null,
+		string $output_method = 'echo'
+	): ?array {
+		return SMF\ServerSideIncludes::boardNews(
+			$board,
+			$limit,
+			$start,
+			$length,
+			$output_method
+		);
+	}
+	function ssi_recentEvents(int $max_events = 7, string $output_method = 'echo'): ?array
+	{
+		return SMF\ServerSideIncludes::recentEvents($max_events, $output_method);
+	}
+	function ssi_checkPassword(int $id = null, string $password = null, bool $is_username = false): bool
+	{
+		return SMF\ServerSideIncludes::checkPassword($id, $password, $is_username);
+	}
+	function ssi_recentAttachments(int $num_attachments = 10, array $attachment_ext = [], string $output_method = 'echo'): ?array
+	{
+		return SMF\ServerSideIncludes::recentAttachments($num_attachments, $attachment_ext, $output_method);
+	}
+	/**
+	 * End
+	 * SMF\ServerSideIncludes
+	 *
+	 * Begin
+	 * SMF\Session
+	 */
+	function loadSession(): void
+	{
+		SMF\Session::load();
+	}
+	/**
+	 * End
+	 * SMF\Session
+	 *
+	 * Begin
+	 * SMF\TaskRunner
+	 */
+	function CalculateNextTrigger(string|array $tasks = [], bool $force_update = false): void
+	{
+		SMF\TaskRunner::calculateNextTrigger($tasks, $force_update);
+	}
+	/**
+	 * End
+	 * SMF\TaskRunner
+	 *
+	 * Begin
+	 * SMF\Theme
+	 */
+	function loadTheme(int $id = 0, bool $initialize = true)
+	{
+		return SMF\Theme::load($id, $initialize);
+	}
+	function loadEssentialThemeData(): void
+	{
+		SMF\Theme::loadEssential();
+	}
+	function loadTemplate(string $template_name, string|array $style_sheets = [], bool $fatal = true): bool
+	{
+		return SMF\Theme::loadTemplate($template_name, $style_sheets, $fatal);
```

Comment:
Try to go to Smileys and Message Icons, and you will get "Return value must be of type bool, null returned"

Reply:
Fixed with return type ?bool

I tested it. Please let me know if this does not fix it on your end.